uptane-standard.md

```diff
@@ -445,11 +445,12 @@ To be available to install on clients, all images on the repository MUST have th
 * The size of the image in bytes
 * One or more hashes of the image file, along with the hashing function used
 
-To avoid a replay attack, Targets metadata from the Director repository SHOULD include a vehicle identifier if there are no images included in the targets metadata.
+If there are no images included in the targets metadata, Targets metadata from the Director repository SHOULD include a vehicle identifier 
+in order toavoid a replay attack.
```

Comment:
```suggestion
If there are no images included in the Targets metadata from the Director repository, then the metadata SHOULD include a vehicle identifier in order to avoid a replay attack.
```